common/scripted_triggers/wc_health_triggers.txt

```diff
@@ -76,4 +76,13 @@ wc_can_breath_underwater = {
 		has_trait_with_flag = titanforged_class
 		has_trait_with_flag = breath_underwater
 	}
+}
+does_not_need_food_trigger = {
+	OR = {
+		scope:character = {
```

Comment:
The same issue, no scopes in the trigger. It may be not set.
For example, we will use the trigger in the events, there is no  `scope:character` thus the trigger will break and cause an error.

`scope:character = {` should be gone. The usings in `00_societal_traditions.txt` should go inside `scope:character`